react/components/atoms/button-tab-text/button-tab-text.js

```diff
@@ -65,12 +71,20 @@ const styles = StyleSheet.create({
         paddingHorizontal: 4,
         alignItems: "center"
     },
+    buttonTabTextCompact: {
+        paddingVertical: 9,
+        paddingHorizontal: 16
+    },
     text: {
         marginTop: Platform.OS === "ios" ? 4 : 0,
         fontFamily: baseStyles.FONT,
         fontSize: 16,
-        letterSpacing: 0.25,
-        textAlign: "center"

```

Comment:
Why?

Reply:
Didn't see any difference with and without. Checked in both iOS and Android